WordPress/Sniffs/NamingConventions/ValidVariableNameSniff.php

```diff
@@ -57,6 +57,15 @@ class WordPress_Sniffs_NamingConventions_ValidVariableNameSniff extends PHP_Code
 	);
 
 	/**
+	 * Mixed-case variables used by WordPress.
+	 *
+	 * @var array
+	 */
+	public $wordpress_mixed_case_vars = array(
+		'EZSQL_ERROR',
```

Comment:
As the `$php_reserved_vars` and `$whitelisted_mixed_case_member_var_names` properties look like they were never intended to be overloadable, I would have no problem breaking backward compatibility in this case.
If someone would currently be overloading them, they are effectively overwriting our defaults which would be a pretty bad idea.